src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -1,225 +1,128 @@
 import { ArtistSeriesHeader_artistSeries } from "v2/__generated__/ArtistSeriesHeader_artistSeries.graphql"
 import { ContextModule } from "@artsy/cohesion"
-import styled from "styled-components"
-import { unitlessBreakpoints } from "@artsy/palette"
-import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { TopContextBar } from "v2/Components/TopContextBar"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
 
 interface ArtistSeriesHeaderProps {
   artistSeries: ArtistSeriesHeader_artistSeries
 }
 
-interface ArtistsInfoProps {
-  // @ts-expect-error STRICT_NULL_CHECK
-  artist: ArtistSeriesHeader_artistSeries["artists"][0]
-  contextOwnerId: string
-  contextOwnerSlug: string
-}
-
-const ArtistInfo: React.FC<ArtistsInfoProps> = props => {
-  /* Displays artist name, avatar and follow button. We currently assume
-     that an artist series will have one artist. */
-  const { artist } = props
+const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = ({
+  artistSeries: {
+    title,
+    descriptionFormatted,
+    artists,
+    image,
+    artworksCountMessage,
+  },
+}) => {
+  if (artists === null) return null
+  const [artist] = artists
+  if (artist === null) return null
 
-  return (
-    <EntityHeader
-      smallVariant
-      name={artist.name}
-      imageUrl={artist.image?.cropped?.src}
-      href={artist.href}
-      FollowButton={
-        <FollowArtistButton
-          artist={artist}
-          contextModule={ContextModule.featuredArtists}
-          render={({ is_followed }) => {
-            return (
-              <Clickable
-                data-test="followArtistButton"
-                textDecoration="underline"
-              >
-                <Text variant="text">
-                  {is_followed ? "Following" : "Follow"}
-                </Text>
-              </Clickable>
-            )
-          }}
-        />
-      }
-    />
-  )
-}
-
-const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
   return (
     <>
-      <Media greaterThanOrEqual="sm">
-        <ArtistSeriesHeaderLarge {...props} />
-      </Media>
-
-      <Media lessThan="sm">
-        <ArtistSeriesHeaderSmall {...props} />
-      </Media>
-    </>
-  )
-}
-
-const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
-  const {
-    artistSeries: {
-      title,
-      descriptionFormatted,
-      artists,
-      image,
-      artworksCountMessage,
-      internalID,
-      slug,
-    },
-  } = props
-  return (
-    <>
-      <Box m={2}>
-        <Flex alignItems="center" justifyContent="center" position="relative">
-          <Flex position="absolute" left={0}>
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            {artists.length && (
-              <ArtistInfo
-                contextOwnerId={internalID}
-                contextOwnerSlug={slug}
-                // @ts-expect-error STRICT_NULL_CHECK
-                artist={artists[0]}
+      <TopContextBar>
```

Comment:
Worked with Jonathan to figure this one out — So now we've got a common `TopContextBar` — we'll probably be adding this back to the Show page and a few others.

Reply:
Mind throwing a screenshot of what this is here?

Reply:
![](https://static.damonzucconi.com/_capture/LxBVfjDT.png)

Just this — Very minor thing.